src/modules/xiv/defaultSnippets/jobSnippets/ast/ast.json

```diff
@@ -0,0 +1,55 @@
+{
+	"snippets": [
+	    {
+            "name": "Earthly Star",
+            "description": "Demonstrate Earthly Star's Giant Dominance Stellar Explosion",
+            "versions": [
+                {
+                    "name": "v0",
+                    "description": "Earthly Star as of 7.05",
+                    "actions": [
+                        {
+                            "action": "Earthly Star"
```

Comment:
should we pre-pull? (I don't think we support it, so no, but sounds like a good future enhancement)